tools/ingest-converter/src/main/java/org/logstash/ingest/JsUtil.java

```diff
@@ -0,0 +1,39 @@
+package org.logstash.ingest;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+
+final class JsUtil {
+
+    private JsUtil() {
+        // Utility Class
+    }
+
+    /**
+     * Sets up a {@link ScriptEngine} for a given file loaded after `ingest-shared.js`.
+     * @param file File to set up {@link ScriptEngine} for
+     * @return {@link ScriptEngine} for file
+     */
+    public static ScriptEngine engine(final String file) {
```

Comment:
They should all be isolated and shouldn't have any dependencies except for the `shared.js` file.